core/src/main/java/com/datastrato/gravitino/storage/relational/utils/POConverters.java

```diff
@@ -303,4 +305,88 @@ public static List<SchemaEntity> fromSchemaPOs(List<SchemaPO> schemaPOs, Namespa
         .map(schemaPO -> POConverters.fromSchemaPO(schemaPO, namespace))
         .collect(Collectors.toList());
   }
+
+  /**
+   * Initialize TablePO
+   *
+   * @param tableEntity TableEntity object
+   * @return TablePO object with version initialized
+   */
+  public static TablePO initializeTablePOWithVersion(
+      TableEntity tableEntity, TablePO.Builder builder) {
+    try {
+      return builder
+          .withTableId(tableEntity.id())
+          .withTableName(tableEntity.name())
+          .withAuditInfo(JsonUtils.anyFieldMapper().writeValueAsString(tableEntity.auditInfo()))
+          .withCurrentVersion(1L)
+          .withLastVersion(1L)
+          .withDeletedAt(0L)
```

Comment:
Can we not hardcode 0L here, instead we'd better define a constant to describe it.

Reply:
Done